### ISSUE TITLE
gutenprint: New version fixes critical issue with some Canon printers

### ISSUE THREAD

**Issue by Moredread** [NixOS/nixpkgs#27869]
## Issue description

The gutenprint printer drivers for some Canon printers had wrong definitions for A4 paper sizes, which caused issues. From the NEWS file in the 5.2.13 version where the issue was fixed.

```
* Changes from 5.2.12

  1) Critical bugfix

     * Correct mis-defined paper type that collided with standard
       A4 paper.  This resulted in some Canon and PCL printers
       mishandling A4 paper.
```

Would this be a candidate for cherry-picking on the 17.03 channel?

See my PR #27867 for the new version.

**Comment by Moredread** [comment]
BTW, what's the policy for updates to stable channels? I've looked for it, but haven't found anything.

**Comment by FRidh** [comment]
Bugfix releases can indeed be backported.

**Comment by FRidh** [comment]
Merged to master and pushed to release-17.03. Thanks.
